subworkflows/local/pairalign_m2m/main.nf

```diff
@@ -109,6 +109,8 @@ workflow PAIRALIGN_M2M {
 
     emit:
 
+    train = LAST_TRAIN.out.param_file
+    tsv = LAST_TRAIN.out.tsv
```

Comment:
Please rename the channel `multiqc`; I think that other workflows or modules use that name too.

Reply:
done